float32[] laser
float32 distance_to_goal
float32 angle_to_goal
